README.md

```diff
@@ -34,12 +32,12 @@ packages:
 
 To run:
 
-	# start up python
-	> python
+    # start up python
+    > python
         ... (startup messages) ...
-	>>> from pyds9 import *
-	>>> print ds9_targets()
-	>>> d = DS9()
+    >>> from pyds9 import *
+    >>> print ds9_targets()
+    >>> d = DS9()
```

Comment:
There are data files [here](https://github.com/ericmandel/js9data/blob/master/fits), and the the [s100.fits](https://github.com/ericmandel/js9data/blob/master/fits/s100.fits) is pretty small (23Kb) and useful (its pixel value encodes the x,y coords).